documents/motions/02.2022.1 - Definitions.md

```diff
@@ -84,7 +80,7 @@ Within WCA we use the following definitions for terminology.
     </tr>
     <tr>
       <td>Country</td>
-      <td>A self-governing geographical area of the world recognized as an independent state by international law and international governmental bodies. <br> The Regulations contain the accepted list of countries.</td>
+      <td>Any country, state, territory or a part of territory recognised by the international community, in conformity with the Olympic Charter and the IOC regulations. <br> The Regulations contain the accepted list of countries.</td>
```

Comment:
```suggestion
      <td>Any country, state, territory or a part of territory recognized by the international community, in conformity with the Olympic Charter and the IOC regulations. <br> The Regulations contain the accepted list of countries.</td>
```

Correct me if I'm wrong, but US English tends to use "z" in similar words.

Reply:
Ah yes, the danger of Australians writing in American English is that we know how to spell "correctly".